packages/errors/src/react/icon-loader.ts

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+// @ts-ignore
+import camelize from 'camelize'
+// @ts-ignore
+import importAll from 'import-all.macro'
+
+const imported = importAll.sync('./icons/*.dist.js')
+
+const convertToIconId = (filePath: string) => {
+  const trimmed = filePath.replace('./icons/', '').replace('-icon.dist.js', '')
```

Comment:
Do we generate .dist.js anymore?  Or are they .dist.ts files?

Reply:
We use dist.js here.  I'm not sure it matters.  They are svgr-generated files.

Reply:
svgr has a typescript output option...but since we don't export the icons i'm not sure it matters